nncf/torch/quantization/layers.py

```diff
@@ -290,6 +291,9 @@ def add_quantization_point(self, qp_id: QuantizationPointId, qp: PTQuantizationP
 
 
 class BaseQuantizer(nn.Module, StatefullModuleInterface, ABC):
+
+    eps: float
```

Comment:
Could you explain the motivation for this change? Why does the current implementation is wrong?

Reply:
After remove torch2 backend it no need, removed